tests/unit_tests/device/cuda/detector_cuda.cpp

```diff
@@ -96,3 +99,50 @@ TEST(detector_cuda, detector) {
         EXPECT_EQ(cylinders_host[i] == cylinders_device[i], true);
     }
 }
+
+TEST(detector_cuda, detector_alignment) {
+    // Memory resources
+    vecmem::host_memory_resource host_mr;
+    vecmem::cuda::managed_memory_resource mng_mr;
+    vecmem::cuda::device_memory_resource dev_mr;
+
+    // Helper object for performing memory copies to CUDA devices
+    vecmem::cuda::copy cuda_cpy;
+
+    // Create toy geometry in host memory
+    auto [det_host, names_host] = build_toy_detector(host_mr);
+
+    // Copy static detector data (including the initial set of transforms) to
+    // the device Use synchronous copy and fixed size buffers
+    auto det_fixed_buff = detray::get_buffer(det_host, dev_mr, cuda_cpy);
+
+    // Construct an "aligned" transform store
+    using test_algebra = test::algebra;
+    using scalar = dscalar<test_algebra>;
+    using point3 = dpoint3D<test_algebra>;
```

Comment:
How about this? Inspired by the changes in unit tests from #893

Reply:
Sure!